### PR TITLE
Don't modify kernel cmdline if secure boot is enabled.

### PR DESCRIPTION
**Pull request by kewu1992**
When secure boot is enabled, kernel command line can not be
modified. This causes the VM into a reboot loop.
This change checks secure boot and fail gracefully when it is enabled.

### CODE REVIEW

File: cos-gpu-installer-docker/entrypoint.sh [hunk 1]
```diff
@@ -136,6 +140,12 @@ configure_kernel_module_locking() {
   fi
 
   if [ "${#sed_cmds[@]}" -gt 0 ]; then
+      # Check secure boot before try to modify kernel cmdline.
+      if is_secure_boot_enabled; then
+        echo "enabled"
```
Comment (arnav-kansal) [inline cos-gpu-installer-docker/entrypoint.sh]: nit: `s/enabled/Secure boot is enabled` or something along those lines maybe?

File: cos-gpu-installer-docker/entrypoint.sh [hunk 2]
```diff
@@ -110,6 +110,10 @@ reboot_machine() {
   echo b > /proc/sysrq-trigger
 }
 
+is_secure_boot_enabled() {
+  sudo dmesg | grep -q 'Secure boot enabled'
```
Comment (arnav-kansal) [inline cos-gpu-installer-docker/entrypoint.sh]: can we run `sudo` inside this env?

Comment (kewu1992) [reply cos-gpu-installer-docker/entrypoint.sh]: Ah, good catch!

File: cos-gpu-installer-docker/entrypoint.sh (hunk 1 above)
Comment (kewu1992) [reply cos-gpu-installer-docker/entrypoint.sh]: I just deleted the line. It's not necessary because we have the error message the next line.

**Review by arnav-kansal — APPROVED** [review]

**Pull request merged**